include/Foundation/NSDateComponents.h

```diff
@@ -48,4 +47,6 @@ FOUNDATION_EXPORT_CLASS
 @property (getter=isLeapMonth) BOOL leapMonth;
 - (NSInteger)valueForComponent:(NSCalendarUnit)unit;
 - (void)setValue:(NSInteger)value forComponent:(NSCalendarUnit)unit;
+- (NSInteger)week;
+- (void)setWeek:(NSInteger)v;
```

Comment:
It's just to align it with the OSX class which does not have a property but the two very propertylike methods.  Though it's a bit silly, it makes more sense to align it with the reference platform rather than have it be different and either modify our analysis tools or flag the code for a deprecated method, especially since this change doesn't affect functionality at all.

---
In reply to: [74354479](https://github.com/Microsoft/WinObjC/pull/868#discussion_r74354479) [](ancestors = 74354479)